TestDB.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+import sys; sys.path.append('../')
+import faker
+
+class TestUser(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(self):
+        self.faker = faker.Faker()
+        faker = self.faker
+
+    def test_save_customer(self):
+        customer = Customer()
+        customer.first_name = faker.first_name()
+        customer.last_name = faker.last_name()
+        customer.date_created = faker.date()
+        user = User()
```

Comment:
If this instance is required for all of the tests in this suite (TestUser), then create a `setUp` method that'll run before each test.

```python
def setUp(self):
  self.user = User()
```